### ISSUE TITLE
misuse of aria-label

### ISSUE THREAD

**Issue by greatgraphicdesign** [validator/validator#1399]
I'm using a div for a hamburger menu button because it isn't valid HTML to put divs inside in a button. This isn't misuse of aria-label, or is it?

```
Warning: Possible misuse of aria-label. (If you disagree with this warning, [file an issue report](https://github.com/validator/validator/issues/new) or send e-mail to www-validator@w3.org.)

From line 898, column 3; to line 898, column 73

button>↩  <div id="icon_menu_container" aria-label="toggle main menu visibility">↩    <
```

**Comment by stevefaulkner** [comment]
@greatgraphicdesign this may be helpful: https://html5accessibility.com/stuff/2020/11/07/not-so-short-note-on-aria-label-usage-big-table-edition/

`<div>` has an implicit `role=generic` which is prohibited from naming https://w3c.github.io/aria/#namefromprohibited

**Comment by greatgraphicdesign** [comment]
Ah. Thanks for the helpful link. I changed it to an anchor tag, and now it passes.
